deployments/_deployments_log_file.json

```diff
@@ -1013,62 +1017,6 @@
       }
     }
   },
-  "WormholeFacet": {
```

Comment:
Why are WormholeFacet and CelerCircleBridgeFacet log entries removed here? Doesnt seem right to me